manifests/plugin/gutterball/config.pp

```diff
@@ -0,0 +1,17 @@
+# gutterball config
+class katello::plugin::gutterball::config(
+    $foreman_user  = 'foreman',
+    $foreman_group = 'foreman',
+    $foreman_plugins_dir = '/etc/foreman/plugins',
+    $protocol = 'https',
+    $port = '8443',
+    $path = 'gutterball/',
```

Comment:
I think we typically defer to leading slashes instead of trailing slashes for URL paths.